taxi/urls.py

```diff
@@ -37,14 +42,35 @@
         ManufacturerDeleteView.as_view(),
         name="manufacturer-delete",
```

Comment:
The same issue here, you should use `-` to split words in URL identification parameter `name`, not the `_`. So, `manufacturer-delete` should be `manufacturer_delete`.